### PR TITLE
Admin show villages

### PR DESCRIPTION
**Pull request by diddka**
Added admin functions to approve, reject and delete villages

### CODE REVIEW

File: src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/EthnicityVillageDTO.java [hunk 1]
```diff
@@ -20,8 +20,9 @@ public class EthnicityVillageDTO {
     private Long villageId;
     private Long ethnicityId;
     private Boolean status;
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
-    private LocalDateTime dateUpload;
+//    @JsonSerialize(using = LocalDateTimeSerializer.class)
+//    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+//    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
```
Comment (Nadezhda358) [inline src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/EthnicityVillageDTO.java:25]: remove commented annotations

File: src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/VillageDTO.java [hunk 2]
```diff
@@ -33,7 +35,16 @@ public class VillageDTO {
...
     private List<String> images;
 
+//    private List<ObjectVillageDTO> objectVillages;
+//    private List<VillageLivingConditionDTO> villageLivingConditions;
+//    private List<VillageAnswerQuestionDTO> villageAnswerQuestions;
+//    private List<EthnicityVillageDTO> ethnicityVillages;
+//    private List<VillageGroundCategoryDTO> villageGroundCategories;
+////    private List<VillageImageDTO> villageImages;
+////    private List<VillageLandscapeDTO> villageLandscapes;
+//    private List<VillagePopulationAssertionDTO> villagePopulationAssertions;
```
Comment (Nadezhda358) [inline src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/VillageDTO.java:48]: remove commented fields

**Review by Nadezhda358 — CHANGES_REQUESTED** [review]

File: src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/VillageDTO.java (hunk 2 above)
Comment (diddka) [reply src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/VillageDTO.java]: ok

**Review by Nadezhda358 — APPROVED** [review]

File: src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/EthnicityVillageDTO.java (hunk 1 above)
Comment (diddka) [reply src/main/java/com/ludogorieSoft/villagelifefrontend/dtos/EthnicityVillageDTO.java]: ok

**Pull request merged**